index.html

```diff
@@ -86,7 +86,7 @@ <h2>Tonic</h2>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
-                    <button class="project-btn" type="button">See Project</button>
+                    <button class="project-btn btn" type="button" data-btn-id="1">See Project</button>
```

Comment:
- [ ] Kindly know that the projects section should be created dynamically with JavaScript for this milestone🙄, as it enables you to remove all hard-coded content and it makes it easier to add more projects later.😊😎

Reply:
Please, can you explain more what do you mean?

Reply:
Please, can you explain more what do you mean?